rules/internal/utils/extension.go

```diff
@@ -65,6 +65,26 @@ func GetResponseType(m *desc.MethodDescriptor) *desc.MessageDescriptor {
 	return typ
 }
 
+// GetOutputOrLROResponseMessage returns the OutputType if the response is
+// not an LRO, or the ResponseType otherwise.
+func GetOutputOrLROResponseMessage(m *desc.MethodDescriptor) *desc.MessageDescriptor {
```

Comment:
Maybe just `GetMethodResponseType`? I kind of feel like the existing `GetResponseType` should be renamed `GetOperationResponseType` and then we can use `GetResponseType` for the OutputOrOperation helper...perhaps that's another PR though.

Reply:
I made the refactor since at least for my ide, it's a one-command change. 

I'll add it in the commit message too.